DFC.App.ExploreCareers.Repository.CosmosDb/CosmosRepository.cs

```diff
@@ -0,0 +1,226 @@
+﻿using DFC.App.ExploreCareers.Data.Models;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Documents.Linq;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Net;
+using System.Threading.Tasks;
+using DFC.Compui.Cosmos.Contracts;
+
+namespace DFC.App.ExploreCareers.Repository.CosmosDb
+{
+    [ExcludeFromCodeCoverage]
+    public class CosmosRepository<T> : Data.Contracts.ICosmosRepository<T>
```

Comment:
You do not  need to define this class yourself as it is defined in the Cosmos nuget